pkg/proxy/ipvs/proxier.go

```diff
@@ -1708,9 +1718,11 @@ func (proxier *Proxier) cleanLegacyService(atciveServices map[string]bool, curre
 	}
 
 	for _, addr := range unbindIPAddr.UnsortedList() {
-		err := proxier.netlinkHandle.UnbindAddress(addr, DefaultDummyDevice)
-		if err != nil {
-			glog.Errorf("Failed to unbind service from dummy interface, error: %v", err)
+		if found := proxier.netlinkHandle.IsDummyDeviceExist(DefaultDummyDevice); found {
```

Comment:
Is it really necessary to check the exist of dummy device here? Since we periodically ensure dummy device exist in sync proxy rules loop.

Reply:
you mean `EnsureDummyDevice()` function? i also confused about it. This error accur in other place, i will paste this log  later.

Reply:
Can we parse the error message and just ignore the not found error?